### PR TITLE
Fix #44

### LINKED ISSUE #44: Refactor all integration tests to be independent one from each other
# 🙋 feature request

Refactor all integration tests to be independent one from each other

## 🤔 Expected Behavior

If you change some test this should not affect any other.

### PR DESCRIPTION
**Pull request by tempora-mutantur**

### CODE REVIEW

File: test/extension.test.ts [hunk 1]
```diff
@@ -18,8 +18,8 @@ import * as path from 'path';
 const checkCompletionItemsForSpecificPosition = (
     cursorPositionForComponent: vscode.Position,
     proposals: Array<vscode.CompletionItem>,
-    itemsShouldExist: boolean = true,
-    fileToOpenInWorkspace: string = 'index.jsx'
+    fileToOpenInWorkspace: string = 'index.jsx',
+    itemsShouldExist: boolean = true
```
Comment (kv4sha) [inline test/extension.test.ts]: Name of a bool variable should start with "is, can, has"

File: testWorkspace/ComponentRequiredPropTest.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
```
Comment (kv4sha) [inline testWorkspace/ComponentRequiredPropTest.jsx]: import for PropTypes

File: testWorkspace/ComponentShapePropTest.jsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
```
Comment (kv4sha) [inline testWorkspace/ComponentShapePropTest.jsx]: the same as above

File: testWorkspace/TriggerSuggestionInsideComponentAttributeTest.jsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
```
Comment (kv4sha) [inline testWorkspace/TriggerSuggestionInsideComponentAttributeTest.jsx]: import for PropTypes

File: test/extension.test.ts (hunk 1 above)
Comment (tempora-mutantur) [reply test/extension.test.ts]: I'll do this in a separate commit because I didn't add this in this PR

**Pull request merged**